copy.go

```diff
@@ -67,6 +67,9 @@ type CopyGraphOptions struct {
 	// OnCopySkipped will be called when the sub-DAG rooted by the current node
 	// is skipped.
 	OnCopySkipped func(ctx context.Context, desc ocispec.Descriptor) error
+	// FindSuccessors finds the successors of the current node.
+	// If FindSuccessors is not provided, a default function will be used.
+	FindSuccessors func(ctx context.Context, fetcher content.Fetcher, desc ocispec.Descriptor) ([]ocispec.Descriptor, error)
```

Comment:
Will this be better and be consistent iwth `FindPredecessors`?
```suggestion
	FindSuccessors func(ctx context.Context, src content.Storage, desc ocispec.Descriptor) ([]ocispec.Descriptor, error)
```